src/bin/common/solving_loop.ml

```diff
@@ -470,12 +490,27 @@ let main () =
         handle_option loc name value;
         st
 
+      | {contents = `Get_model; _ } ->
+        if Options.get_interpretation () then
+          match State.get sat_env_key st with
+          | Some sat_env -> SAT.get_model sat_env; st
+          | None ->
+            (* TODO: add the location of the statement. *)
+            Printer.print_wrn "No model produced.";
```

Comment:
This is a response to a command; it should be `(error "No model produced.")` according to the smtlib standard. It's not very important, but we should open an issue.